styles.css

```diff
@@ -726,4 +738,139 @@ iframe{
 .responsive-image{
     width: 30vw;
     height: auto;
+}
+
+/* updated */
+
+.input-container,
+.input-label-pair {
+    display: flex;
+    flex-direction: column;
+}
+
+.input-container{
+    background-color: var(--white);
+    padding: 2rem;
+    border-radius: 1rem;
+    box-shadow: 5px 5px 15px var(--dark-grey);
+}
+
+.input-component-preview{
+    justify-content: center;
+    gap: 2rem;
+}
+
+.input-label {
+    font-weight: 500;
+    font-size: 1.25rem;
+    margin: 0.25rem 0;
+}
+
+.input-label-pair input {
+    outline: none;
```

Comment:
don't use outline : none rather than use outline : 0

Reply:
thanks for the review, have made the changes suggested above.